webpack/index.template.html

```diff
@@ -13,7 +13,36 @@
     <!-- End Google Tag Manager -->
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- HTML Meta Tags -->
     <title>Simularium</title>
+    <meta name="description"
+        content="The Simularium Viewer makes it easy to share and interrogate interactive 3D visualizations of biological simulation trajectories and related plots directly in a web browser.">
+    
+    <!-- Facebook Meta Tags -->
+    <meta property="og:title" content="Simularium" />
+    <meta property="og:url" content="https://simularium.allencell.org/">
+    <meta property="og:image" content="../src/assets/meta-tag.png">
```

Comment:
Does this relative URL get transformed at all in the build process, and/or does the image get put in the build directory? The image may not appear in the hosted site if it's dependent on navigating out of the build directory and into src.

Reply:
That's a good question. When I was first looking at this I thought we should put the asset into s3 and use an absolute URL but @meganrm said a relative URL was the more modern way to handle it.

I think this is a good catch. I assumed it was working because my preview looking okay, but I'm looking for the `png` in the build folder and not seeing it.

I think with our build rules we will need to do an import to ensure that the `png` comes along for the ride, and the easiest way to do that will be to inject these tags progamatically. Going to poke around looking at options for that.

Reply:
RE: doing this dynamically/in React: because web-crawlers don't execute javascript it seems likely we will either need a static URL directly in the HTML or have to implement server-side rendering, or get the back end more involved in this. For SSR there are some libraries but that seems like a lift for this feature.

I think just uploading the necessary images to S3 seems like the most straightforward way to handle this. Would you agree @frasercl ?